modules/eks/cluster/variables.tf

```diff
@@ -1,6 +1,7 @@
 variable "region" {
   type        = string
   description = "AWS Region"
+  nullable    = false
```

Comment:
`region` is required. Only set `nullable = false` for inputs that are optional and where `null` is not a valid option.

```suggestion
```